libraries/AP_BoardConfig/px4_drivers.cpp

```diff
@@ -31,12 +30,15 @@
 #include <nuttx/arch.h>
 #include <spawn.h>
 
+#include <AP_HAL_PX4/CAN.h>
+#include <AP_UAVCAN/AP_UAVCAN.h>
```

Comment:
CAN driver should be on not depending on UAVCAN but it has some UAVCAN reference.